debug/examples/shop-consumables-list.json5

```diff
@@ -55,16 +55,16 @@
             {
                 type: "image",
                 name: "itemIcon",
-                assetKey: "${icon}",
-                position: { align: "left", offsetX: 35, offsetY: 0 },
```

Comment:
is it not good to keep offset for fine tuning position?

Reply:
it is fine tuned

Reply:
ah nevermind, I see it's still in the calculation :)